app/templates/index.hbs

```diff
@@ -0,0 +1,127 @@
+<div class='jumbo' style="
+    max-height: 1200px ;
+    background-color: rgb(197, 214, 230);
+    padding-bottom: 40px ;
+    background: url('/assets/images/bg6.jpg');
+    background-size: cover ;
+    ">
+  <div>
+    <img height="20%" width="20%"
+      style="display:block; margin-left:auto; margin-right:auto"
+      src='assets/images/main_logo.png'>
+  </div>
+
+  <div class="container">
+    <div class="row" style="display: flex ; justify-content: center ;">
+      <div class="col-xs-offset-1 col-xs-10" >
+        {{input class="form-control" width='100%' value=searchString key-up='typing' placeholder='Search . . . ' }}
+      </div>
+      <div class="col-xs-1">
+        <a href="" {{action 'search'}}>
+          <i style="font-size: 150%; margin-top: 5px" class="fa fa-search"></i>
+        </a>
+      </div>
+    </div>
+    <br>
+    <div class="row" style="display: flex ; justify-content: center ;">
+      <a href="/discover" style="font-family: 'Open Sans' , sans-serif ; font-size: 12px ;">Advanced Search</a>
+    </div>
+  </div>
+
+
+  <div class="row" style="padding-top: 40px; display: flex ; justify-content: center ; flex-wrap: wrap ;">
+    <div class="tile" style=" margin: 1% ;">
```

Comment:
Could this line go in the component template since it is only being used with the `homepage-button-box`?